doc/source/tracking/traffic/traffic_data_mgmt.py

```diff
@@ -0,0 +1,51 @@
+import matplotlib.pyplot as plt
+import os
+import pandas as pd
+import subprocess
+
+cwd = os.getcwd()
+
+trafficpath = cwd + "/doc/source/tracking/traffic/"
+clonesfn = "clones.csv"
+viewsfn = "views.csv"
+defaultpath = "{}/{}/".format(cwd, "traffic")
+
+def update_csv(fn, string):
+    try:
+        existing = pd.read_csv(trafficpath + fn)
+        new = pd.read_csv(defaultpath + fn)
+        updated = new.merge(
+            existing, how="outer", on=["_date", "total_" + string, "unique_" + string]
+        )
+    except FileNotFoundError:
+        updated = pd.read_csv(defaultpath + fn)
+
+    updated.sort_values("_date", ignore_index=True).to_csv(
+        trafficpath + fn, index=False
+    )
+
+    return updated
+
+
+clones = update_csv(clonesfn, "clones")
+views = update_csv(viewsfn, "views")
```

Comment:
Simplified function to just take one 'string' parameter, and use [f-strings](https://realpython.com/python-f-strings/).

```suggestion
trafficpath = f"{cwd}/doc/source/tracking/traffic/"
defaultpath = f"{cwd}/traffic/"


def update_csv(string):
    try:
        existing = pd.read_csv(trafficpath + f"{string}.csv")
        new = pd.read_csv(defaultpath + f"{string}.csv")
        updated = new.merge(
            existing, how="outer", on=["_date", f"total_{string}", f"unique_{string}"]
        )
    except FileNotFoundError:
        updated = pd.read_csv(defaultpath + string)

    updated.sort_values("_date", ignore_index=True).to_csv(
        trafficpath + f"{string}.csv", index=False
    )

    return updated


clones = update_csv(string="clones")
views = update_csv(string="views")
```